translations/stripes-authorization-components/en.json

```diff
@@ -58,6 +58,7 @@
   "applications.totalSelected": "Total selected: {count}",
   "applications": "Applications",
   "applications.notAvailable": "Select application is not available",
+  "applications.unselect.warning": "By unselecting {appNames}, {capabilityCount} capabilities and {capabilitySetCount} capability sets will also be unselected.  Are you sure you’d like to proceed?",
```

Comment:
l10n is not correct here. Numeric (`capabilityCount`, `capabilitySetCount`) and plural (`capabilities`, `capability sets`) values must be labeled as such.